src/vmware/azext_vmware/tests/latest/test_workload_network_scenario.py

```diff
@@ -0,0 +1,111 @@
+        self.assertEqual(dnsServiceUpdate['name'], 'dnsService1')
+
+        dnsServiceDelete = self.cmd('az vmware workload-network dns-service delete --resource-group {rg} --private-cloud {privatecloud} --dns-service-id {dns_service_id}').output
+        self.assertEqual(len(dnsServiceDelete), 0)
+
+        dnsZoneList = self.cmd('az vmware workload-network dns-zone list --resource-group {rg} --private-cloud {privatecloud}').get_output_in_json()
+        self.assertEqual(len(dnsZoneList), 1, 'count expected to be 1')
+
+        dnsZoneGet = self.cmd('az vmware workload-network dns-zone show --resource-group {rg} --private-cloud {privatecloud} --dns-zone-id {dns_zone_id}').get_output_in_json()
+        self.assertEqual(dnsZoneGet['name'], 'portMirroring1')
+
+        dnsZoneCreate = self.cmd('az vmware workload-network dns-zone create --resource-group {rg} --private-cloud {privatecloud} --dns-zone-id {dns_zone_id} --display-name {display_name} --domain {domain} --dns-server-ips {dns_server_ips} --source-ip {source_ip} --dns-services {dns_services} --revision {revision}').get_output_in_json()
+        self.assertEqual(dnsZoneCreate['name'], 'dnsZone1')
+
+        dnsZoneUpdate = self.cmd('az vmware workload-network dns-zone update --resource-group {rg} --private-cloud {privatecloud} --dns-zone-id {dns_zone_id} --display-name {display_name} --domain {domain} --dns-server-ips {dns_server_ips} --source-ip {source_ip} --dns-services {dns_services} --revision {revision}').get_output_in_json()
+        self.assertEqual(dnsZoneUpdate['name'], 'dnsZone1')
+
+        dnsZoneDelete = self.cmd('az vmware workload-network dns-zone delete --resource-group {rg} --private-cloud {privatecloud} --dns-zone-id {dns_zone_id}').output
+        self.assertEqual(len(dnsZoneDelete), 0)
+
+        portMirroringList = self.cmd('az vmware workload-network port-mirroring list --resource-group {rg} --private-cloud {privatecloud}').get_output_in_json()
+        self.assertEqual(len(portMirroringList), 1, 'count expected to be 1')
+
+        portMirroringGet = self.cmd('az vmware workload-network port-mirroring show --resource-group {rg} --private-cloud {privatecloud} --port-mirroring-id {port_mirroring_id}').get_output_in_json()
+        self.assertEqual(portMirroringGet['name'], 'portMirroring1')
+
+        # Uncomment these unit tests once swagger is fixed
+
+        # portMirroringCreate = self.cmd('az vmware workload-network port-mirroring create --resource-group {rg} --private-cloud {privatecloud} --port-mirroring-id {port_mirroring_id} --display-name {display_name} --direction {direction} --source {source} --destination {destination} --revision {revision}').get_output_in_json()
```

Comment:
You should be able to run these locally.

Reply:
I'm not able to run these locally because the direction parameter only accepts `"INGRESS, EGRESS, BIDIRECTIONAL"`, instead of `INGRESS` `EGRESS` or `BIDIRECTIONAL`. I can change the direction parameter for the test cases to `"INGRESS, EGRESS, BIDIRECTIONAL"`, but that will break once the swagger is updated. The current commented out implementation will work once the swagger is correct.

Reply:
I'd prefer to have it run now. The value can be fixed when we upgrade to 2021-12-01 API.